src/test-file-server.ts

```diff
@@ -0,0 +1,31 @@
+import { Sequence, Execution } from 'effection';
+import { on } from '@effection/events';
+import { ChildProcess, fork as forkProcess } from '@effection/child_process';
+
+interface TestFileServerOptions {
+  files: [string];
+  manifestPath: string;
+  port: number;
+};
+
+export function* createTestFileServer(orchestrator: Execution, options: TestFileServerOptions): Sequence {
+  // TODO: this should use node rather than ts-node when running as a compiled package
+  let child: ChildProcess = yield forkProcess(
+    './bin/parcel-server.ts',
+    ['-p', `${options.port}`, '--out-file', 'manifest.js', '--global', '__bigtestManifest', options.manifestPath],
+    {
+      execPath: 'ts-node',
```

Comment:
When we ship this to end users, we're going to want to pre-compile the typescript. Should we tag locations like this that will require recompilation, with a comment like a `// @precompile` so that we can find them in the future?

Reply:
Yes, could be a good idea to mark locations that we know will require attention when packaging.